demos/browser/app/meeting/meeting.ts

```diff
@@ -474,6 +475,34 @@ export class DemoMeetingApp implements AudioVideoObserver, DeviceChangeObserver
     });
   }
 
+  getNearestMediaRegion(): void {
+    const supportedMediaRegions: Array<string> = ['ap-northeast-1', 'ap-southeast-1', 'ap-southeast-2', 'ca-central-1', 'eu-central-1', 'eu-north-1', 'eu-west-1', 'eu-west-2', 'eu-west-3', 'sa-east-1', 'us-east-1', 'us-east-2', 'us-west-1', 'us-west-2'];
+    const defaultNearestMediaRegion = "us-east-1";
+    new AsyncScheduler().start(
+      async (): Promise<void> => {
+      try {
+        const nearestMediaRegionResponse = await fetch(
```

Comment:
From @ltrung:
> the join button event handler created in initEventListeners is also an aysnc job which reads from inputRegion. So if this job for some reason completes after the users click Join, it would not work. Maybe its fine and we can just use the default region in that case. What do you think @simmkyu

I think it should be fine to use the default region since it's a demo app. In production, we'll enable the "Join" button only after regions are selected.